templates/dmodash.html

```diff
@@ -6,9 +6,12 @@
 
 <div class="container text-center">
     <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue  DMO Portal</span></h1>
-    <p style="margin-bottom:20px;">
-        <i>Please Select the district to view stats </i><br/>
-  </p>
+        <p style="margin-bottom:20px;">
+            <a href="https://process.keralarescue.in/"><b>Click here to view the rescue dashboard</b></a><br/>
+        </p>
+        <p style="margin-bottom:20px;">
+            <i>Please Select the district to view stats </i><br/>
```

Comment:
`view stats`
@aswins Is this a typo ?

Reply:
Nope. That shows stats. Also that was existing code. Just indented.

Reply:
Done